packages/app/src/components/Blank.cy.tsx

```diff
@@ -0,0 +1,85 @@
+import { initial, initialCT, session, sessionLifecycle, visitFailure } from './Blank'
+import { ROOT_ID } from '@cypress/mount-utils'
+
+describe('initial - e2e', () => {
```

Comment:
Cool, I wonder if we need a recipe in the main docs for how to use Cypress component w/o a framework. How was the experience of developing this using CT once you knew that you could use the runner workflow to do it?

This is how I like to develop components. I also don't usually run with `yarn dev` - just `yarn cypress:open`, that way it's nice and quick (you don't get 500 watchers running every time you save something).